### PR TITLE
debian: drop revision from version, illegal for native-format packages

### PR DESCRIPTION
**Pull request by nabijaczleweli**
Confer:
```
$ dpkg-buildpackage -us -uc
dpkg-buildpackage: info: source package swtpm
dpkg-buildpackage: info: source version 0.5.0-1
dpkg-buildpackage: info: source distribution RELEASED
dpkg-buildpackage: info: source changed by Stefan Berger <stefanb@linux.ibm.com>
dpkg-source: error: can't build with source format '3.0 (native)': native package version may not have a revision
```

### CODE REVIEW

**Comment by stefanberger** [comment]
I suppose you are building it on Debian rather than Ubuntu? On Ubuntu it works as-is but I do remember that there were issues on Debian.

File: debian/changelog [hunk 1]
```diff
@@ -1,4 +1,4 @@
-swtpm (0.5.0-1) RELEASED; urgency=medium
+swtpm (0.5.0) RELEASED; urgency=medium
```
Comment (stefanberger) [inline debian/changelog:1]: Can you please add your Signed-off-by: to the patch description?

And if you could make this a PR against the stable-0.5 branch that would help as well. Thank you!

**Comment by nabijaczleweli** [comment]
I am, indeed, on Debian; both buster and sid yield the same error.

Added signoff, rebased on top of `stable-0.5`, retargetted at `stable-0.5`.

**Comment by stefanberger** [comment]
Thanks!

**Pull request merged**